src/frontends/pytorch/src/translate_session.cpp

```diff
@@ -220,8 +220,15 @@ OutputVector TranslateSession::convert_node(const NodeContext& context) {
         OPENVINO_DEBUG << "Some exception happened during conversion of node of type: " << context.get_op_type()
                        << '\n';
     }
-    // Create PtFrameworkNode for everything that wasn't able to be converted normally
-    return make_framework_node(context);
+    try {
+        // Create PtFrameworkNode for everything that wasn't able to be converted normally
+        return make_framework_node(context);
```

Comment:
Really interesting that creation of FW node can throw an exception. How can it happen?:)

Reply:
If body of Loop lets say fails for some reason on validation, but it is a good point, maybe better solve that during fw node creation